### PR TITLE
Provide node version

### PR DESCRIPTION
**Pull request by excelsiooor**

### CODE REVIEW

**Review by Andrew1412Belous — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,8 +1,5 @@
 {
-  "name": "@dream/team",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "@dream-team/monorepo",
```
Comment (AmateurBoy) [inline package.json:2]: name full shit

**Review by AmateurBoy — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (DannyNickson) [reply package.json:2]: Agree with @AmateurBoy.
We can probably name it like "dream-project" or "blue-cherry-project"

File: package.json [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
   "scripts": {
     "prelint": "npm run prettier",
     "lint": "npx turbo run lint"
```
Comment (DannyNickson) [inline package.json]: We can add a new commands like "up" or "setup" to automaticly download all require files

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+v18.12.0
```
Comment (DannyNickson) [inline .nvmrc]: Add extra line please
Follow the prittier suggestions

**Review by DannyNickson — CHANGES_REQUESTED** [review]
Left a few comments :smiling_imp:

**Review by User84265 — APPROVED** [review]
looks amazing, i really like this code style, good job

**Review by Andrew1412Belous — APPROVED** [review]

File: .nvmrc (hunk 3 above)
Comment (excelsiooor) [reply .nvmrc]: done

File: package.json [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-  "main": "index.js",
+  "name": "@dream-team/monorepo",
   "scripts": {
-    "prelint": "npm run prettier",
-    "lint": "npx turbo run lint"
+    "setup": "yarn install",
+    "setup:flat": "yarn install --flat",
+    "setup:pure": "yarn install --pure-lockfile",
+    "setup:update": "yarn install --update-checksums",
+    "clean:modules": "npx rimraf node_modules"
```
Comment (DannyNickson) [inline package.json:8]: Do we need to add Makefile to simplify work with repo? 
Suggesting to add TODO if such thing is acceptable. 
```suggestion
    // TODO: New Issue: Add Makefile to simplify work with repo 
    "clean:modules": "npx rimraf node_modules"
```

File: package.json [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
   },
   "workspaces": [
     "packages/*",
```
Comment (DannyNickson) [inline package.json:11]: Also, a quick question: Will Docker run this? If so, we 100% need to add Makefile or extend package.json

**Review by DannyNickson — CHANGES_REQUESTED** [review]
Left a few comments to clarify project architecture :smiling_imp:

File: package.json (hunk 5 above)
Comment (excelsiooor) [reply package.json:11]: I think it should be handled in 100% separately on the Docker stage.

File: package.json (hunk 4 above)
Comment (excelsiooor) [reply package.json:8]: Comments are not permitted in JSON.
But I'm sure we'll get to it on the docker stage.

**Review by DannyNickson — APPROVED** [review]
Good job! Thanks for clarification

**Review by Oleksandr-Filo — APPROVED** [review]
LGTM

**Pull request merged**